code-challenges.js

```diff
@@ -1,69 +1,127 @@
+    // Concantenation will be used to combine the two arrays. Since it is an accessor it will not alter the original values
+    // Another built-in method is needed to return the length of the combined arrays.
+    // The result should show 9 as the expected output since there are 5 values in the first array and 4 in the second array.
 
-const temperature3 = 212
-// Expected output: "212 is at boiling point"
+const padresWorldSeriesRuns = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+    console.log(padresWorldSeriesRuns.length);
+        // Expected output: 9
 
-// Pseudo code:
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+// 3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+    const currentCohort = "Charlie 2023"
 
-// Pseudo code:
+// Pseudo Code:
+    // I need to declare another variable (currentCohort) to represent the string that will be reversed. Using reverse on the variable currentCohort is not possible since the data type is not compatible with the method.
+    // To start, it is necessary to turn a string into an array. Adding in quotations inside its grouping operator indexes the elements in specific location. 
+    // Now that the is now separated into different elements inside an array, the reverse method can now be used to reverse the index of the elements.
+    // The join method will be used to connect the separated elements, but adding in quotations inside its grouping operator removes the commas that separate the elements. 
+    // The expected output should show the reversed "3202 eilrahC"
 
-const currentCohort = "Charlie 2023"
-// Expected output: "3202 eilrahC"
+console.log(currentCohort.split("").reverse().join(""));
+    // Expected output: "3202 eilrahC"
 
-// --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-// Pseudo code:
+// 4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
+    const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 
-const givenValue1 = 42
-// Expected output: 7
+        const givenValue1 = 42
+        const index1 = numberOfConnections.lastIndexOf(givenValue1)
+        console.log(index1);
+        // Expected output: 7
 
-const givenValue2 = 10
-// Expected output: 8
+        const givenValue2 = 10
+        const index2 = numberOfConnections.lastIndexOf(givenValue2)
+        console.log(index2);
+        // Expected output: 8
```

Comment:
No need to create variables `index1` and `index2`, you can just log the logic they are holding instead.